src/App.jsx

```diff
@@ -27,46 +27,90 @@ function App() {
     });
```

Comment:
Remove package-lock.json since I use yarn, to install yarn use `npm i yarn -g`,
Then you can install packages using yarn install...
yarn run dev...
If you want to install third party packages use yarn add vuejs (example)